src/Core/Checkout/Document/Renderer/OrderDocumentCriteriaFactory.php

```diff
@@ -44,7 +44,7 @@ public static function create(array $ids, string $deepLinkCode = ''): Criteria
         $criteria->getAssociation('transactions')->addSorting(new FieldSorting('createdAt'));
         $criteria->getAssociation('deliveries')->addSorting(new FieldSorting('createdAt'));
 
-        if ($deepLinkCode !== '') {
+        if ($deepLinkCode !== null && $deepLinkCode !== '') {
```

Comment:
as PHPStan already noted, the variable cannot be `null` at this point. it is typed as string in the method parameters. 

so maybe this is not the right place to fix that? :thinking:

Reply:
a test where you show how the issue is reproducible would be helpful in this case

Reply:
Ah yes, thats weird, didn't see that thanks! Will write a test for it.